src/ast-types.ts

```diff
@@ -14,10 +14,32 @@ interface Location {
   }
 }
 
+export const commentTypes = ['BlockComment', 'LineComment'] as const
+
+export type CommentTypeString = typeof commentTypes[number]
```

Comment:
Can we do this instead, or do you need the `commentTypes` value?

```suggestion
export type CommentTypeString = 'BlockComment' | 'LineComment'
```